Prefer seed from uri over load (#2010)
neo4j/docs-operations: modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc

DROP DATABASE movies DUMP DATA
----

In Neo4j, dumps can be stored in the directory specified by the xref:configuration/configuration-settings.adoc#config_server.directories.dumps.root[`server.directories.dumps.root`] setting (by default, the path for storing dumps is xref:configuration/file-locations.adoc#data[`<neo4j-home>/data/dumps`]).

> Dumps can be used to create databases using the xref:clustering/databases.adoc#cluster-seed-uri[Seed from URI approach].

Reviewer: ```suggestion
You can use dumps to create databases through the xref:clustering/databases.adoc#cluster-seed-uri[Seed from URI approach].
```

The option `DESTROY DATA` explicitly requests the default behavior of the command.

[NOTE]
====
The dumps produced by `DUMP DATA` are equivalent to those produced by xref:backup-restore/offline-backup.adoc[`neo4j-admin database dump`]. 

> They can also be restored using the xref:backup-restore/restore-dump.adoc[`neo4j-admin database load`] command.

Reviewer: ```suggestion
You can also restore them using the xref:backup-restore/restore-dump.adoc[`neo4j-admin database load`] command.
```
